src/main.js

```diff
@@ -65,6 +66,7 @@ export default function createApp({
 	// Provide locale to all components
 	// TODO: use this to set locale in VueI18n
 	Vue.prototype.$locale = locale;
+	Vue.prototype.$browser = browser;
```

Comment:
Adds this object globally
<img width="461" alt="Screen Shot 2021-02-01 at 4 26 01 PM" src="https://user-images.githubusercontent.com/187937/106537679-b396f000-64af-11eb-9cb5-0ffbf6bd44a4.png">

Reply:
Perhaps a different name than $browser? Any thoughts?

Reply:
I'm ok with `$browser`, but if we were to change it I think `$device` might be a good option as it's an object with all the info about the device being used to view the site, not just the browser